index.js

```diff
@@ -184,12 +184,14 @@ async function buildPlatforms () {
             }
 
             // Ensure the correct enabled state for Click to Load entities.
-            const clickToLoadSettings = platformConfig?.features?.clickToPlay?.settings
```

Comment:
FWIW I think the lack of check for `key === 'clickToPlay'` here was a bug. We were updating the disabled/enabled status for Click to Load entities again for every feature.

Reply:
FWIW we talked about impact of this bug with @kzar  and this wasn't a bug that was causing trouble, just being unoptimal and it only started causing trouble when Dave added [key] logic. Anyway, we are all good here.